modules/__tests__/Media-test.js

```diff
@@ -77,34 +76,34 @@ describe('A <Media>', () => {
     });
 
     describe('and a children element', () => {
-      it('renders its child', () => {
+      it('does not render its child', () => {
         const element = (
           <Media query="">
             <div>hello</div>
           </Media>
         );
 
-        ReactDOM.render(element, node, () => {
-          expect(node.firstChild.innerHTML || '').not.toMatch(/hello/);
+        renderStrict(element, node, () => {
+          expect(node.innerHTML || '').not.toMatch(/hello/);
         });
       });
     });
 
     describe('and a children function', () => {
-      it('renders its child', () => {
+      it('renders according to a non match', () => {
         const element = (
           <Media query="">
             {matches => (matches ? <div>hello</div> : <div>goodbye</div>)}
           </Media>
         );
 
-        ReactDOM.render(element, node, () => {
+        renderStrict(element, node, () => {
           expect(node.firstChild.innerHTML).toMatch(/goodbye/);
         });
       });
     });
 
-    describe('and a render function', () => {
+    describe.only('and a render function', () => {
```

Comment:
```suggestion
    describe('and a render function', () => {
```
I suspect this `.only` is not intended to stay here.